enterprise/api/v1/mixins.py

```diff
@@ -51,22 +51,29 @@ def update_course(self, course, enterprise_customer, enterprise_context):
         Returns:
             (dict): Updated course metadata
         """
+        enterprise_utm_context = {
```

Comment:
I think it would be better to add utm context in the `enterprise_context` dict.

```
enterprise_context.update({
    'utm_medium': 'enterprise',
    'utm_source': enterprise_customer.name
})
``` 
That way it will be available to all other methods.

Reply:
Actually, I was thinking the same but `enterprise_context` is in use by some other methods [e.g.](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/mixins.py#L85 )that I think, at that point do not require to have UTM information in query parameters.

Reply:
I agree with @saleem-latif - it makes sense to me that this is generally available because we want to add this tracking to any link we are providing to customers to which they will potentially send traffic.

Reply:
@brittneyexline If we update the `enterprise_context` dict with the `UTM` information then we will get the `UTM` information in course as we are updating the course with `enterprise_context` values [HERE](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/mixins.py#L66)
So we will get the `UTM` information in [Catalog Response](https://github.com/edx/edx-enterprise/blob/master/tests/test_enterprise/api/test_views.py#L486)

Reply:
alright, i see what you are saying. i think that is fine as long as we are updating all of the urls as described in the other comments.